scripts/variantstore_cromwell_tests/import_array_manifest_test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ImportArrayManifest.extended_manifest_csv":"/home/travis/build/broadinstitute/gatk/src/test/resources/org/broadinstitute/hellbender/tools/variantdb/arrays/tiny_manifest.csv",
+  "ImportArrayManifest.manifest_schema_json":"/home/travis/build/broadinstitute/gatk/scripts/variantstore_wdl/schemas/manifest_schema.json",
+  "ImportArrayManifest.project_id":"broad-dsde-dev",
```

Comment:
Is this just copied from elsewhere?  Seems like there should be a gatk-test google project...

Reply:
This is the GATK test project AFAIK. It's used in the BigQueryUtils tests (in GATK).